README.md

```diff
@@ -25,55 +26,27 @@ The Busola also includes React and Angular libraries:
 - [`npm`](https://www.npmjs.com/): >= 6.4.0
 - [`node`](https://nodejs.org/en/): >= 12.0.0
 
-## Installation
-
-1. Install [Kyma](https://kyma-project.io/docs/master/root/kyma/#installation-install-kyma-locally) as a backing service for your local instance of Busola. Make sure you import certificates into your operating system and mark them as trusted. Otherwise, you cannot access the applications hosted in the `kyma.local` domain.
-
-2. Install Busola dependencies. To install dependencies for the root and all UI projects, and prepare symlinks for local libraries within this repository, run the following command:
+## Usage with NPX
```

Comment:
I intentionally put this section **before** the `Installation` one to emphasise one doesn't have to perform any manual installation steps to use the NPX mode. Since the `Usage` section is below the `Installation` in the template, I'm afraid one may follow the Readme top-to-down, then realise "Oh great, I could've skipped this time-consuming step...". 
Of course I can do that change if you still think I should 😄